src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/QTClusterFinder.h

```diff
@@ -112,10 +113,24 @@ namespace OpenMS
 
     /// Map to store which grid features are next to which clusters
     typedef OpenMSBoost::unordered_map<
-              OpenMS::GridFeature*, std::vector<QTCluster*> > ElementMapping;
+              OpenMS::GridFeature*, std::unordered_set<Size> > ElementMapping;
+
+    /// Handles for the clusters in the index
+    typedef boost::heap::fibonacci_heap<QTCluster>::handle_type Handle;
```

Comment:
Not sure about this, because the handles are a way to access clusters inside the heap and not the heap in some way. Therefore I would prefer `ClusterHandle`.

Reply:
Its a heap. that should be reflected in the name?!

Reply:
Naming it `HandleForClustersInHeap` is not a good option in my opinion. We can either name the handle after what it manages or after the datastructure which contains the objects it manages. Knowing what it manages is the more valuable information when using the handle.

Reply:
Removed the typedef completely. Not it's `Heap::handle_type`. I think this is a good solution.